src/config/config.ts

```diff
@@ -558,6 +604,116 @@ export class Config implements IConfig {
   protected get isProd() {
     return isProd()
   }
+
+  private getLookupId(id: string): string {
+    if (this.commandIndex.has(id)) return id
+    if (this.permutationIndex.hasValid(id)) return this.permutationIndex.getValid(id)!
+    return id
+  }
+
+  private loadCommands(plugin: IPlugin) {
+    for (const command of plugin.commands) {
+      if (this.commandIndex.has(command.id)) {
+        const priotitized = this.determinePriority([this.commandIndex.get(command.id)!, command])
+        this.commandIndex.set(priotitized.id, priotitized)
```

Comment:
Spelling & make variable name a little more descriptive.
```suggestion
        const prioritizedCommand = this.determinePriority([this.commandIndex.get(command.id)!, command])
        this.commandIndex.set(prioritizedCommand.id, prioritizedCommand)
```